schema/bom-1.6.xsd

```diff
@@ -2173,9 +2173,12 @@ limitations under the License.
 
     <xs:complexType name="componentEvidenceType">
         <xs:sequence>
-            <xs:element name="identity" minOccurs="0" maxOccurs="1">
+            <xs:element name="identity" minOccurs="0" maxOccurs="unbounded">
                 <xs:annotation>
-                    <xs:documentation>Evidence that substantiates the identity of a component.</xs:documentation>
+                    <xs:documentation>Evidence that substantiates the identity of a component. The identify may be an
```

Comment:
better suiting for XML:
Evidence that substantiates the identity of a component.

Support for specifying identify as a single object was introduced in CycloneDX v1.5. "unbound" was introduced in v1.6. It is RECOMMENDED that all implementations are aware of "unbound".

Reply:
Good call. Corrected.